modules/ROOT/pages/bundling-plugins.adoc

```diff
@@ -15,39 +15,155 @@ include::partial$plugin-files/plugin-file-index.js.adoc[]
 
 Example syntax for including the example "<plugincode>" plugin in a bundle using `content.js`for bundling:
 
-[cols='1,1,4']
+[cols='1,2,3']
 |===
 |Module Syntax |Source |Example
 
-.2+|ES6+
-|npm
+.4+|ES6+
+|npm (community plugins)
 a|
 [source, js]
 ----
 import 'tinymce/plugins/<plugincode>';
 ----
 
-|`.zip`
+|npm (premium plugins)
+a|
+[source, js]
+----
+import 'tinymce-premium/plugins/<plugincode>';
+----
+
+|`.zip` (community plugins)
 a|
 [source, js]
 ----
 import '../tinymce/plugins/<plugincode>';
 ----
 
-.2+|Common JS
-|npm
+|`.zip` (premium plugins)
+a|
+[source, js]
+----
+import '../tinymce-premium/plugins/<plugincode>';
+----
+
+.4+|Common JS
+|npm (community plugins)
 a|
 [source, js]
 ----
 require('tinymce/plugins/<plugincode>');
 ----
 
-|`.zip`
+|npm (premium plugins)
+a|
+[source, js]
+----
+require('tinymce-premium/plugins/<plugincode>');
+----
+
+|`.zip` (community plugins)
 a|
 [source, js]
 ----
 require('../tinymce/plugins/<plugincode>');
 ----
+
+|`.zip` (premium plugins)
+a|
+[source, js]
+----
+require('../tinymce-premium/plugins/<plugincode>');
+----
 |===
 
+[NOTE]
+====
+When using `+import tinymce from 'tinymce'+`, import only the individual premium plugins from `+tinymce-premium+` (e.g., `+import 'tinymce-premium/plugins/advcode'+`). Do not import the `+tinymce-premium+` package. The plugins automatically register with the {productname} instance.
+====
+
+[[using-premium-plugins]]
+== Using premium plugins
+
+After installing the `+tinymce-premium+` package, you need to configure the plugins in your editor. There are two main approaches:
+
+=== Bundling (Recommended)
+
+For most modern applications, bundling premium plugins is recommended. This includes the plugin code directly in your application bundle.
+
+. Import the premium plugins you need:
++
+[source,js]
+----
+import 'tinymce-premium/plugins/advcode';
+import 'tinymce-premium/plugins/licensekeymanager';
+import 'tinymce-premium/plugins/tinycomments';
+----
+
+. Add the plugins to your editor configuration:
++
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  license_key: 'T8LK:...', // Your commercial license key
+  plugins: 'advcode tinycomments',
+  // ... other configuration
+});
+----
+
+For complete bundling examples, see the <<Contents>> section below.
+
+=== Using external_plugins (Non-bundling)
+
+When bundling is not available or you want to lazy-load plugins, use the `+external_plugins+` option:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  license_key: 'T8LK:...', // Your commercial license key
+  external_plugins: {
+    'advcode': '/node_modules/tinymce-premium/plugins/advcode/plugin.min.js',
```

Comment:
Consider `<path_to_premium_plugins>` or something here too, instead of `/node_modules/`? This example isn't directly linked to installing via NPM - could be ZIP or anything either, and that way it's more generalised.